invenio_oaiserver/query.py

```diff
@@ -154,8 +154,8 @@ def items(self):
                         'id': result['_id'],
                         'json': result,
                         'updated': datetime.strptime(
-                            result['_source']['_oai']['updated'],
-                            '%Y-%m-%dT%H:%M:%SZ'
+                            result['_source']['_updated'][:19],
```

Comment:
Perhaps better to use ``arrow.get(result['_source']['_updated'])`` from the arrow library to get a timezone aware timestamp.